src/solver/conv_ck_igemm_fwd_bias_activ_fused.cpp

```diff
@@ -411,15 +411,18 @@ bool ConvCKIgemmFwdBiasActivFused::IsApplicable(const FusionContext& ctx,
         return false;
     const auto conv_problem = fdesc_problem.GetConvProblem(0, conv::Direction::Forward);
 
-    if(conv_problem.IsTensorsCasted())
-        return false;
```

Comment:
[Recommendation] It's better not to move code that hasn't changed (here and in other places too)

Reply:
So I moved these to order the checks by a guess of cost. Expensive checks should come after cheaper checks.